ios/Helio.xcodeproj/xcshareddata/xcschemes/HelioStaging.xcscheme

```diff
@@ -60,9 +60,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "YES">
-      <Testables>
-      </Testables>
+      shouldUseLaunchSchemeArgsEnv = "YES"
+      shouldAutocreateTestPlan = "YES">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new attribute `shouldAutocreateTestPlan` has been added and set to "YES". This means Xcode will automatically create a test plan if one doesn't exist. If this is not the desired behavior, consider removing this line.
```diff
+   shouldAutocreateTestPlan = "YES"
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->